### ISSUE TITLE
Calling propTypes directly - Warning thrown

### ISSUE THREAD

**Issue by IamTheHttp** [react-bootstrap/react-bootstrap#2610]
My previous ticket was closed too promptly I believe:
https://github.com/react-bootstrap/react-bootstrap/issues/2605

Minimal code to reproduce:
```
import { FormControl } from 'react-bootstrap';
import React from 'react';
import ReactDOM from 'react-dom';
ReactDOM.render(
  <FormControl/>,
  document.getElementById('app')
);
```

In two different React Environments, I'm getting two different errors.(both with version 0.31.0)

Using React 15.2.1 and ReactDOM 15.2.1

> Failed prop type: Calling PropTypes validators directly is not supported by the `prop-types` package. Use `PropTypes.checkPropTypes()` to call them. (In Gylphicon)

Using React 15.1.0 and ReactDOM 15.0.0

> Failed propType: Calling PropTypes validators directly is not supported by the `prop-types` package. Use `PropTypes.checkPropTypes()` to call them.

> Failed Context Types: Calling PropTypes validators directly is not supported by the `prop-types` package. Use `PropTypes.checkPropTypes()` to call them

**Comment by IamTheHttp** [comment]
My bad! related to old react versions.

When upgrading to react 15.3.0 this issue is resolved.

**Comment by gaearon** [comment]
Please see https://github.com/reactjs/prop-types/issues/38.

This can also occur as a false positive caused by production version of `prop-types` generating a mistaken warning when development version of React is calling it.

**It is fixed in `prop-types@15.5.10`.**

You might need to rebuild the UMD bundle of the component that depends on `prop-types` (probably this repo) to see the changes.

**Comment by IamTheHttp** [comment]
Thanks, that was an issue in one of our other projects!
